bin/zowe-configure-instance.sh

```diff
@@ -256,6 +256,18 @@ fi
 chmod -R 755 ${INSTANCE}
 chmod -R 755 ${INSTANCE_DIR}/bin
 
+# Go through Zowe build-in components and see they need to be configured for current instance
+component_list="jobs-api files-api api-catalog discovery gateway caching-service apiml-common-lib explorer-ui-server explorer-jes explorer-mvs explorer-uss"
```

Comment:
what about `app-server` and `zss`?

Reply:
We will add app-server and zss once those packages are converted to follow manifest definition. Currently it's backward compatible, we had converted apiml and explorer apis/uis, so we just start from here.